csharp/Platform.Setters/Setter[TResult, TDecision].cs

```diff
@@ -6,21 +6,39 @@
 namespace Platform.Setters
 {
     /// <summary>
-    /// <para>Represents implementation for an setter that allows you to set a passed value as the result value. This setter implementation has additional methods that, simultaneously with setting the result value, return <typeparamref name="TDecision"/> values indicating true or false.</para>
-    /// <para>Представляет реализацию для установщика, который позволяет установить переданное ему значение в качестве результирующего значения. В этой реализации установщика есть дополнительные методы, которые одновременно с установкой результирующего значения возвращают значения типа <typeparamref name="TDecision"/>, обозначающие истину или ложь.</para>
+    /// <para>Represents a setter that allows to set a passed value as the result. This setter variant has additional methods that, simultaneously with setting the result, return <typeparamref name="TDecision"/> values indicating true or false.</para>
+    /// <para>Представляет реализацию для установщика, который позволяет установить переданное ему значение в качестве результата. В этой реализации установщика есть дополнительные методы, которые одновременно с установкой результата возвращают значения типа <typeparamref name="TDecision"/>, обозначающие истину или ложь.</para>
     /// </summary>
-    /// <typeparam name="TResult"><para>The type of result value.</para><para>Тип результирующего значения.</para></typeparam>
-    /// <typeparam name="TDecision"><para>The type of value which will be used to make the decision.</para><para>Тип значения на основе которого будет приниматься решение.</para></typeparam>
+    /// <typeparam name="TResult">
+    /// <para>The type of a result value.</para>
+    /// <para>Тип результирующего значения.</para>
+    /// </typeparam>
+    /// <typeparam name="TDecision">
+    /// <para>The type of true and false values.</para>
+    /// <para>Тип значений истина и ложь.</para>
+    /// </typeparam>
     public class Setter<TResult, TDecision> : SetterBase<TResult>
     {
         private readonly TDecision _trueValue;
+        
         private readonly TDecision _falseValue;
 
         /// <summary>
-        /// <para>Initializes a new instance of the Setter class using the passed-in value as the default result value.</para>
-        /// <para>Инициализирует новый экземпляр класса Setter, используя переданные значения trueValue, falseValue, defaultValue в качестве результирующего по умолчанию.</para>
+        /// <para>Initializes a new instance of the <see cref="Setter{TResult, TDecision}"/> class using the passed-in <paramref name="trueValue"/>, <paramref name="falseValue"/>, <paramref name="defaultValue"/> values as the default result.</para>
+        /// <para>Инициализирует новый экземпляр класса <see cref="Setter{TResult, TDecision}"/>, используя переданные значения <paramref name="trueValue"/>, <paramref name="falseValue"/>, <paramref name="defaultValue"/> в качестве результата по умолчанию.</para>
```

Comment:
```suggestion
        /// <para>Инициализирует новый экземпляр класса <see cref="Setter{TResult, TDecision}"/>, используя переданные значения <paramref name="trueValue"/>, <paramref name="falseValue"/>, <paramref name="defaultValue"/>.</para>
```

Reply:
This ending can confuse user into thinking that trueValue and falseValue are possible values for a Result property. But they are not.